maths/aliquot_sum.py

```diff
@@ -0,0 +1,39 @@
+from typing import Union
+
+
+def aliquot_sum(input_num: int) -> Union[int, str]:
+    """
+    Finds the aliquot sum of an input integer, where the
+    aliquot sum of a number n is defined as the sum of all
+    natural numbers less than n that divide n evenly. For
+    example, the aliquot sum of 15 is 1 + 3 + 5 = 9. This is
+    a simple O(n) implementation.
+    @param input_num: a positive integer whose aliquot sum is to be found
+    @return: the aliquot sum of input_num, if input_num is positive.
+    Otherwise, return the string "Please enter a positive integer"
+    Wikipedia Explanation: https://en.wikipedia.org/wiki/Aliquot_sum
+    
+    >>> aliquot_sum(15)
+    9
+    >>> aliquot_sum(6)
+    6
+    >>> aliquot_sum(-1)
+    'Please enter a positive integer'
+    >>> aliquot_sum(12)
+    16
+    >>> aliquot_sum(19)
+    1
```

Comment:
Please add `aliquot_sum(0)` and `aliquot_sum(1.7)`.

Reply:
Is there a way to directly modify the pull request without making a new one? I'm kind of new to Git.

Reply:
Never mind, I think I found out how